### PR TITLE
Shivani61 patch 5

### PR DESCRIPTION
**Pull request by shivani61**
I added all the text and images provided by keith for android studio plugin and made a small change on docs.esper.io home page..  the android studio plugin has changed the esper device kit section.. and it has been placed above esper virtual device on the docs.esper.io dev docs ..

### CODE REVIEW

**Comment by kirananto** [comment]
@shivani61  

jpg !== JPG
Files names — ergo their extensions — are case sensitive.
Allowed extensions are jpeg/jpg/gif/png

Renaming it would help solve the issue here.

**Comment by shivani61** [comment]
changed all jpg files from uppercase to lowercase. please retry building.

File: docs/home/esperplugin.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+## Requirements
+Before you start with this guide, make sure you have Android Studio installed, version 181 or greater. 
+To verify your Android Studio go to Help → About. The version number is circled below:
+
+
+![](./assets/OLD_DASHBOARD/1_asp_AndroidStudioAbout.jpg)
+
+In case you have a version before 181, simply upgrade your Android Studio to the latest version. The latest available version can be installed by clicking on Help → Check for Updates... and then following the subsequent prompts.
+You will also need an account on an Esper Endpoint. Have your Endpoint name, username, and password available. If your company doesn’t have an Esper Endpoint, sign up for an Esper Trial account here.
+
```
Comment (tarun91vas) [inline docs/home/esperplugin.md]: `sign up for an Esper Trial account here.` This is not pointing to any link

File: docs/home/esperplugin.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# Esper Android Studio Plugin
+<div class="avg-reading-time">Average reading time : {{ $page.readingTime.text.replace(' read', '') }}</div>
+
+
+Esper Plugin for Android Studio
```
Comment (tarun91vas) [inline docs/home/esperplugin.md]: I think new line is missing here. Preview shows it in same line.

File: docs/home/esperplugin.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+The Esper Plugin for Android Studio enables developers to upload APKs directly from Android Studio to an Esper Endpoint. Once uploaded to Esper you can use the Esper APIs, SDK, or CLI from Android Studio Terminal to provision devices with the APK for testing or deployment purposes. Or you can go to the Esper Console and work with your APK from there.
+This guide describes the steps to take to install the Esper Plugin into Android Studio.
+
+## Requirements
+Before you start with this guide, make sure you have Android Studio installed, version 181 or greater. 
+To verify your Android Studio go to Help → About. The version number is circled below:
```
Comment (tarun91vas) [inline docs/home/esperplugin.md]: (nit) It is good to have `help -> About` and similar references as bold.

File: docs/home/esperplugin.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+
+The Plugin is now configured to your Endpoint for this Project.
+
+Now once you are ready to upload an .apk file to your Endpoint, simply go to Build → Upload APK to Esper:
+
+![](./assets/OLD_DASHBOARD/5_asp_Build.jpg)
+
+
+In the dialog that appears, choose the .apk to upload to your Endpoint, and then hit OK:
```
Comment (tarun91vas) [inline docs/home/esperplugin.md]: Should we address it as `.apk` or just apk file ?

Comment (shivani61) [reply docs/home/esperplugin.md]: .apk is ok i guess

**Pull request merged**